src/client/components/Nav/index.js

```diff
@@ -65,7 +65,6 @@ const Nav = ({
           target="_blank"
           rel="noopener roreferrer"
           onClick={() => {
-            setCurrentTab(constants.TABS.COURSE_RESERVES);

```

Comment:
Why is setCurrentTab no longer used here?

Reply:
It was never supposed to be there, I think it got merged on accident at some point by either one of us